src/checkers/cwe_248.ml

```diff
@@ -0,0 +1,65 @@
+open Bap.Std
+open Core_kernel.Std
+
+let name = "CWE248"
+let version = "0.1"
+
+(* Print the findings to the log *)
+let print_uncatched_exception block_tid ~tid_map =
+   Log_utils.warn
```

Comment:
FYI, you could use BAP native logging facilities. Just `include Self()` and you will get `debug`, `info`, `warning`, and `error` functions.